### ISSUE TITLE
As a CIF administrator, I want to see a cleaner diff section in Amendments/General Revisions, so that I the clearer context of the changes is hidden but easily accessible

### ISSUE THREAD

**Issue by nanyangpro** [bcgov/cas-cif#1573]
#### Description:

This is a cleaner version of #1569 preferred by users where all contextual/help text of why the user sees a certain change to a field is hidden behind a hover over tooltip.

[- Design: Cleaner Version preferred by users](https://www.figma.com/file/PzOY8RyQnVXnjZJdSv2AqQ/CIF---Internal-User-Portal?node-id=7647%3A912&t=9ZUtWsS7W7FVvhtv-1)

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by nanyangpro** [comment]
AC was drafted, pending further refinements and Dev Checklist from the developers. Happy to provide clarifications.

**Comment by Sepehr-Sobhani** [comment]
Is tooltip implementation also part of this card? shouldn't we have a separate card for that?

**Comment by pbastia** [comment]
see #1270

**Comment by nanyangpro** [comment]
@pbastia: To be discussed in Backlog Refinement May 2.

**Comment by BCerki** [comment]
#1569 is probably going to split out the contextual help AC, so this ticket could become the entire implementation of contextual help

**Comment by nanyangpro** [comment]
#1742 was create as a stop gap for #1573 & #1575. @dleard

**Comment by pbastia** [comment]
At this point this is a lot of work for value that hasn't been quite proven given #1742. I suggest iceboxing

**Comment by nanyangpro** [comment]
Good call, @pbastia - This ticket only makes sense when #1659 is in place. Given #1742 was suggested as a stop gap solution for #1569. This ticket (#1573) can be ice-boxed.
